docs/commands/splitmulti.md

```diff
@@ -125,6 +125,6 @@ After this pipeline, the info field "AC" in `1kg.split.vcf.bgz` will have number
 <div class="cmdsubsection">
 ### <a name="splitmulti_annotations"></a> Annotations:
 
- - `va.wasSplit       Boolean` -- this variant was originally multiallelic 
- - `va.aIndex             Int` -- The original index of this variant in imported line, 0 for imported biallelic sites
-</div>
\ No newline at end of file
+ - `va.wasSplit : Boolean` -- this variant was originally multiallelic
+ - `va.aIndex : Int` -- the original allele index (NB: 0 is the reference) of this variant in imported line, 0 for imported biallelic sites
```

Comment:
original index of this allele...1 for imported biallelic sites
Example: 1:100:A:T,C => 1:100:A:T with aIndex 1, and 1:100:A:C with aIndex 2